Finjector.Web/Controllers/FolderController.cs

```diff
@@ -56,6 +56,37 @@ public async Task<IActionResult> Get(int id)
             return Ok(new { folder, charts });
         }
 
+        [HttpGet("folderSearchList")]
+        [ProducesResponseType(typeof(IEnumerable<Folder>), StatusCodes.Status200OK)]
+        public async Task<IActionResult> FolderSearchList()
+        {
+            var iamId = Request.GetCurrentUserIamId();
+
+            var folders = await _dbContext.Folders
+                // is there a better/more efficient way to do this? -river
+                .Include(f => f.FolderPermissions).ThenInclude(fp => fp.User)
+                .Include(f => f.FolderPermissions).ThenInclude(fp => fp.Role)
+                .Include(f => f.Team).ThenInclude(t => t.TeamPermissions).ThenInclude(tp => tp.User)
+                .Include(f => f.Team).ThenInclude(t => t.TeamPermissions).ThenInclude(tp => tp.Role)

```

Comment:
you shouldn't need any of these includes -- they are only for if you aren't doing a `.Select` projection